### ISSUE TITLE
Erroneous image links in `Modelica.Mechanics.Translational`

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#310]
**Reported by dietmarw on 11 Mar 2010 20:24 UTC**
When converting the image links to "modelica://" URIs I found some illegal and in case of one not functioning image link. 

```diff
-<<IMG SRC=../Images/Translational/PreLoad.png>
+<IMG src=\"modelica://Modelica/Images/Translational/PreLoad.png\">
```
and 
```diff
-<IMG src=\"modelica://Modelica/Images/Translational/PreLoad3.png>\"
+<IMG src=\"modelica://Modelica/Images/Translational/PreLoad3.png\">
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/310

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 11 Mar 2010 20:24 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 11 Mar 2010 20:25 UTC**
Fixed in 0efb8680c3f6d0d8e75f61305bd7ef957524da47.
